src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -426,7 +439,15 @@ tests:
     - label: "Step 10b: Reset Devices to factory defaults"
       PICS: PICS_USER_PROMPT
       verification: |
-          Reset Devices to factory defaults
+          To perform Factory reset on DUT(all-clusters-app)  follow the below step in RPI platform , Pls use equivalent command on the respective platform
+
+          sudo rm -rf /tmp/chip_*
+
+          execute the below mentioned command to put DUT into a commissionable state, Pls use equivalent command on the respective DUT
+
+          step-1:  create a file using touch command , something like touch mytest.txt
+          step-2:  chmod 777 mytest.txt
+          step-3:  launch the app  ./chip-all-clusters-app --KVS ./mytest.txt
```

Comment:
See above about these comments not making sense.

Reply:
Reverted back to the previous version.